scripts/install.sh

```diff
@@ -378,27 +222,10 @@ main() {
 
         set -eE
         install_slack_cli "$@"
-
-        sleep 0.1
-        >&2 echo -e "\x1b[1m⚠️  Warning: Starting on September 1, 2025, Deno will no longer be installed with this script!\x1b[0m"
-        >&2 echo -e "⚠️  Warning: Apps built with Deno should install Deno separately:"
-        >&2 echo -e "⚠️  Warning: https://docs.deno.com/runtime/getting_started/installation/"
-        echo -e ""
-        install_deno
-
-        echo
-        if [ -f "$(command -v code)" ]; then
-                delay 0.1 "📦 Adding editor support for an enhanced experience..."
-                install_deno_vscode_extension
-                if [ $? -eq 0 ]; then
-                        echo -e "✨ Nice! You're all set to start developing!"
-                fi
-        fi
-
         sleep 0.2
-        feedback_message
-        sleep 0.1
         terms_of_service
+        sleep 0.1
+        feedback_message
```

Comment:
Out of scope, but is there a way to toggle these sleep off to improve the CI experience?

Reply:
I don't believe so right now, but this is a good idea to follow up on 🙏

IIRC we've discussed environment variable options but I forget at the moment if we settled on something? I'll also be thinking about this!